integration-tests/chopsticks/src/index.ts

```diff
@@ -0,0 +1,25 @@
+import { connectParachains, connectVertical } from '@acala-network/chopsticks'
+
+import * as SpiritnetNetwork from './network/spiritnet.js'
+import * as PolkadotNetwork from './network/polkadot.js'
+import * as HydraDxNetwork from './network/hydroDx.js'
+import { keysCharlie, keysBob } from './helper.js'
+
+/// Helper function to validate the storage of the chains. The chains will not produce any blocks on demand.
+/// TODO: fix that
+async function spinUpNetwork() {
```

Comment:
So what's the goal of this function?

Reply:
It was quite handsome to see if updating the storage of the chains did work as expected. I could connect to polkadotjs and investigate the chain states.

Reply:
Yes, I get that. So you start the network before running the tests?

Reply:
No, this is not necessary. The problem with the tests was, that the network shuts down once the test cases are over. I could avoid this by using the `.pause` function of the chain network configs, but then I needed to increase the time out of the tests: which was quite annoying.

Reply:
No, respinning the network overwrites the database. It was just used to observe how the storage appears after using `.dev.setStorage`.